package.json

```diff
@@ -27,7 +27,7 @@
     "@download/blockies": "^1.0.3",
     "@ledgerhq/hw-transport-u2f": "^4.41.1",
     "@zxing/library": "^0.12.1",
-    "base-x": "3.0.4",
+    "base-x": "^3.0.5",
```

Comment:
base-x@3.0.5 requires transpilation, I have added it to vue.config.js, need to ensure that it works in some old version of iOS.